docs/sources/guides/whats-new-in-v6-6.md

```diff
@@ -34,12 +34,71 @@ Grafana 6.6 comes with a lot of new features and enhancements:
 
 More details will be added as we're getting closer to the stable release.
 
+## New stat panel
+
+{{< docs-imagebox img="/img/docs/v66/stat_panel_dark.png" max-width="1024px" caption="Stat panel" >}}
+
+This release adds a new panel named `Stat`. This panel is designed to replace the current `Singlestat` as the primary way
+to show big single number panels along with a sparkline. This panel is of course building on our new panel
+infrastructure and option design. So you can use the new thresholds UI and data links. It also supports the same
+repeating feature as Gauge & Bar Gauge panels, meaning it will repeat a separate visualization for every series or row
+in the query result.
+
+Key features:
+
+- Automatic font size handling
+- Automatic layout handling based on panel size
+- Colors based on thresholds that adapt to light or dark theme
+- Data links support
+- Repeats horizontally or vertically for every series, row or column
+
+Here is how it looks in light theme:
+
+{{< docs-imagebox img="/img/docs/v66/stat_panel_light.png" max-width="1024px" caption="Stat panel" >}}
+
+## Auto min max
+
+For the panels Gauge, Bar Gauge & Stat you can now leave the min and max settings empty. Grafana will, in that case, calculate the min & max based on
+all the data.
+
+## New time picker
+
+The time picker has gotten a major design update. Key changes:
+
+- Quickly access the absolute from & to input fields without an extra click.
+- Calendar automatically shows when from or to inputs has focus
+- A single calendar view can be used to select and show the from & to date.
+- Select recent absolute ranges
+
+{{< docs-imagebox img="/img/docs/v66/time_picker_update.png" max-width="700px" caption="New time picker" >}}
+
+## News panel
+
+This panel supports showing RSS feeds as news items. It is used in the updated default home dashboard. Add it to
+your custom home dashboards to keep up to date with Grafana news or switch the default RSS feed to one of your choice.
```

Comment:
Should we add info that feed needs to be a RSS feed? Tried Atom feed and it didn't work https://www.theverge.com/rss/index.xml